bazel/BUILD.jemalloc

```diff
@@ -13,7 +13,13 @@ configure_make(
     copts = ["-fPIC"],
     args = ["-j"],
     out_shared_libs = ["libjemalloc.so"],
-    configure_options = ["--disable-static", "--enable-prof"],
+    # See https://salsa.debian.org/debian/jemalloc/-/blob/c0a88c37a551be7d12e4863435365c9a6a51525f/debian/rules#L8-23
+    # for why we are setting "--with-lg-page" on non x86 hardware here.
+    configure_options = ["--disable-static", "--enable-prof"] +
+        select({
+            "@platforms//cpu:x86_64": [],
```

Comment:
don't we needt oset --with-lg-page=12?

Reply:
No, the existing behavior is fine (on x86, the page size is the same on all the architectures and it will select that on compilation)